### DISCUSSION TITLE
1.8.0 Pre-release

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-AutoUpdate#62] [General]
@Romanitho 
You'll have to create the directory 'mods' too!

**Comment by Romanitho** [comment]
Will be created on your first mod copy (not tested 😅)

**Comment by KnifMelti** [comment]
Nope, the copy doesn't occure until the directory is in place...

**Reply by Romanitho** [reply]
Oh yes, I have double checked. I thought if the folder was not there, it would be created by the install script. Because I cannot have an empty folder on Git.
I can create one with dummy txt file.

**Reply by KnifMelti** [reply]
Or a readme

**Reply by Romanitho** [reply]
I put your readme in the main one. Better for comprehension :)

**Reply by Romanitho** [reply]
Fixed. 1.8.0 preview has the mods folder now

**Comment by KnifMelti** [comment]
I notice that when WAU updating to 1.8.0 (stable) from 1.7.4 it doesn't upgrade the winget version to "Using preview version to resolve issue as system", shouldn't it do that?

**Reply by Romanitho** [reply]
nope. Winget is install if needed only when you run the install script. When we update WAU, we do not pay attention for the Winget version, that is the choice of the end user.
That's why we check on first install only (and from few days we check for the preview version as it fixes one identified bug).
